rapid7_insightvm/komand_rapid7_insightvm/util/resource_helpers.py

```diff
@@ -164,9 +189,9 @@ def validate_user_email(self, email: str) -> None:
         email_regex = re.compile(r'[^@]+@[^@]+\.[^@]+')
 
         if not email_regex.match(email):
-            error = 'ERROR: Email address for user account was not valid!'
-            self.logger.error(error)
-            raise Exception(error)
+            error = 'The email address for user account was not valid!'
+            raise PluginException(cause=error,
```

Comment:
error variable can be removed and the text can be moved directly into the exception